boxoffice/assets/js/views/org.js

```diff
@@ -2,33 +2,79 @@
 var Ractive = require('ractive');
 import {eventBus} from './main_admin.js'
 var NProgress = require('nprogress');
-import {fetch, urlFor, setPageTitle} from '../models/util.js';
+import {Util, fetch, post, formErrorHandler, getFormParameters, urlFor, setPageTitle} from '../models/util.js';
 import {orgTemplate} from '../templates/org.html.js';
 import {SideBarView} from './sidebar.js'
 
 export const OrgView = {
   render: function({org_name}={}) {
+    const DEFAULT = {
+      showForm: true,
+      hideForm: false,
+      empty: ""
+    };
+
     fetch({
       url: urlFor('view', {resource: 'o', id: org_name, root: true})
-    }).then(function({id, org_title, item_collections}){
+    }).then(function({id, org_title, item_collections, form}) {
+      let ICForm = Ractive.extend({
+        isolated: false,
+        template: Util.getFormTemplate(form, 'onFormSubmit(event)'),
+        data: {
+          formId: Util.getElementId(form),
+          formSubmit: DEFAULT.btnEnable
+        },
+        onFormSubmit: function(event) {
+          event.original.preventDefault();
+          let formSelector = '#' + this.get('formId');
+          post({
+            url: urlFor('new', {
+              scope_ns: 'o',
+              scope_id: org_name,
+              resource: 'ic',
+              root: true
+            }),
+            data: getFormParameters(formSelector),
+            formId: formSelector
+          }).done((remoteData) => {
+            orgComponent.unshift('item_collections', remoteData.result.item_collection);
```

Comment:
Are these two lines required since we're navigating to the dashboard directly?

Reply:
Fixed